CHANGELOG.md

```diff
@@ -12,6 +14,25 @@
 - Replaced the example for Java system properties in `specification/configuration.md`.
 - Increase profiling rate from 1s to 10s.
 
+### Configuration
+
+- Change the Kubernetes package management solutions configuration option
+  default for `metricsEnabled` to be `false`.
```

Comment:
AFAIK this is only for Splunk Platform. How about:

```suggestion
- Change the Kubernetes package management solutions configuration option
  default for `splunkPlatform.metricsEnabled` to be `false`.
```

see https://github.com/signalfx/gdi-specification/pull/152